### DISCUSSION TITLE
[Question] MediaElement MKV Support

### DISCUSSION THREAD

**Discussion by mhrastegari** [xamarin/XamarinCommunityToolkit#1233] [Q&A]
can we have mkv support for MediaElement?
or is there a way to support MKV formats with MediaElement that i don't know?

**Comment by pictos** [comment]
@peterfoot do you have any idea about this?

**Reply by peterfoot** [reply]
I don't think it is supported with the built in player controls. I'm not sure what additional work is required to add this.

**Comment by mhrastegari** [comment]
or is there at least a way to make it full screen on android?
iOS and UWP has it why not android? 🤦‍♂️

**Reply by AndreiMisiukevich** [reply]
Fell free to implement it and submit a PR ;)

**Reply by DerekFoulk** [reply]
@mhrastegary77 I am also surprised the Android player lacks full screen functionality and MKV support. Evidently it doesn't resize when you rotate the device either... Android just doesn't get the same amount of love as iOS! I will try and figure it out, then submit a PR for us buddy :)
